tasks/strategy/git.yml

```diff
@@ -1,14 +1,20 @@
 ---
 
-- name: strategy/git > Cached repository
+- name: strategy/git > Update repository
   git:
     repo:           "{{ manala_deploy_strategy_git_repo }}"
-    dest:           "{{ deploy_helper.shared_path ~ '/cached-copy' }}"
+    dest:           "{{ manala_deploy_dir ~ '/repo' }}"
```

Comment:
This would give `/srv/app/repo` instead of `/srv/app/shared/cached-copy`, right?